webapp/.eslintrc.json

```diff
@@ -631,17 +629,31 @@
         "exceptRange": false,
         "onlyEquality": false
       }
+    ],
+    "formatjs/no-multiple-whitespaces": 2,
+    "formatjs/enforce-default-message": 2,
+    "formatjs/no-multiple-plurals": 2,
+    "formatjs/enforce-placeholders": 2,
+    "unused-imports/no-unused-imports": 2,
+    "no-relative-import-paths/no-relative-import-paths": [
+      "error",
+      { "allowSameFolder": true }
     ]
   },
   "overrides": [
     {
       "files": ["**/*.tsx", "**/*.ts"],
-      "parser": "@typescript-eslint/parser",
-      "plugins": ["@typescript-eslint"],
       "extends": "plugin:@typescript-eslint/recommended",
       "rules": {
+        "@typescript-eslint/no-empty-function": 0,
+        "@typescript-eslint/ban-ts-ignore": 0,
```

Comment:
Are we really using `ts-ignore` anywhere?

Reply:
We should remove this disabled rule then, so the linter will complain about it if we introduce it. `ts-ignore` ends up ignoring a whole file when used, which is a pretty error-prone way to use typescript